action.yml

```diff
@@ -142,7 +142,7 @@ runs:
       working-directory: ${{ inputs.base-path }}
       shell: bash
       run: |
-        pip3 install -r zephyr/scripts/requirements.txt
+        west packages pip --install --ignore-venv-check || pip3 install -r zephyr/scripts/requirements.txt
```

Comment:
could you add a comment to drop the fallback case after 4.1 has been phased out? thanks!